package.json

```diff
@@ -98,6 +98,7 @@
     "karma-sourcemap-loader": "^0.3.7",
     "karma-webpack": "^1.7.0",
     "mocha": "^2.2.5",
+    "nodemon": "^1.9.1",
```

Comment:
@nathanmarks you're right. but with  node v5.4.1 (npm v3.7.5) I'm not receiving that warning, if you want I can left it out